src/Raven.Server/Documents/Handlers/Admin/RachisAdminHandler.cs

```diff
@@ -419,13 +419,6 @@ public async Task AddNode()
                                 $"Adding a new node to cluster failed. The new node is already in another cluster. " +
                                 $"Expected topology id: {topologyId}, but we get {nodeInfo.TopologyId}");
                         }
-
-                        if (nodeInfo.CurrentState != RachisState.Passive)
```

Comment:
You are actually removing a check that seems valid, no?
Can't this now cause us to be added to the cluster twice?

Reply:
This check seems to be redundant.
for not adding the same node twice we have this:
https://github.com/ravendb/ravendb/blob/b22d8f611e62bca6b5f055feee3ee69708904554/src/Raven.Server/Documents/Handlers/Admin/RachisAdminHandler.cs#L414